README.md

```diff
@@ -1422,6 +1422,7 @@ _Related: [Wikis](#wikis)_
 - [Snibox](https://snibox.github.io/) - Code snippets manager with attractive tag-oriented interface. ([Demo](https://snibox-demo.herokuapp.com/), [Source Code](https://github.com/snibox/snibox)) `MIT` `Ruby`
 - [Snippet Box](https://github.com/pawelmalak/snippet-box) - Snippet Box is a simple self-hosted app for organizing your code snippets. It allows you to easily create, edit, browse and manage your snippets in various languages. `MIT` `Nodejs`
 - [snipt](https://github.com/nicksergeant/snipt) - Long-term memory for coders. Share and store code snippets. `MIT` `Python`
+- [SnyPy](https://snypy.com) - Open source on-prem code snippet manager. ([Demo](https://app.snypy.com), [Source Code](https://github.com/snypy)) `MIT` `Python`
```

Comment:
Where can I find installation instructions? What are the server side requirements?

Reply:
The setup is not straight forward as it can be done in multiple ways for the Rest API. The easiest setup is using containers and it is documented in this repository: https://github.com/snypy/snypy-docker (The repository is also used to build and push the containers to the github container registry: https://github.com/orgs/snypy/packages)

Reply:
> The setup is not straight forward as it can be done in multiple ways

It's good practice to document at least 1 working setup. But since Docker is the only supported installation method, please change the language/platform tag to `Docker`.

> The Language tag is the main server-side requirement for the software

Reply:
Tag is now set to `Docker`